packaging/cmake/try_compile/pthread_setname_freebsd_test.c

```diff
@@ -0,0 +1,7 @@
+#include <pthread.h>
+#include <pthread_np.h>
+
+int main() {
+  pthread_set_name_np("abc");
```

Comment:
Seems like it is missing the threadid argument

Reply:
Corrected